### DISCUSSION TITLE
More better new library name

### DISCUSSION THREAD

**Discussion by muukii** [FluidGroup/Brightroom#68] [Ideas]
We'd like to get something better completely different name for this library.

Against: https://github.com/muukii/Pixel/discussions/63

**Comment by muukii** [comment]
BrightRoom

**Reply by amith-ranasinghe** [reply]
DotEditor ;)

**Reply by muukii** [reply]
Cool, that looks like an alternative way of "Pixel"!

**Comment by muukii** [comment]
In this industry we can see those names:
Lightroom
Darkroom
